examples/minio/minio-pvc.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   name: minio
   namespace: open-cluster-management-observability
 spec:
-  storageClassName: gp2
+  storageClassName: standard
```

Comment:
@songleo the default storageclass in openshift is `gp2`, so this change will result PV can't be created in e2e env.
I know you need this in KinD env, can we change in script for e2e in KinD?